gripper_xyzrpy: [0.19, 0.0, 0.0, 0.0, 0.0, 0.0]
